test/tests/package.json

```diff
@@ -4,9 +4,9 @@
   "main": "progressive/index.stache!done-autorender",
   "dependencies": {
     "can-component": "^4.0.0",
-	"can-define": "^2.0.0",
-    "can-fixture": "^2.0.0",
-	"can-route": "^4.0.0",
+    "can-define": "^2.0.0",
+    "can-fixture": "^3.1.0",
```

Comment:
This is in `dependencies`… should it have been a major bump?

Reply:
Fixed!